arch/arm/arm64/pl011.inc

```diff
@@ -0,0 +1,50 @@
+#include <arch/pl011.h>
+
+/*
+ * PL011 UART initialization
+ * xb: register which contains the UART base address
+ * c: scratch register number
+ */
+.macro pl011_uart_init xb, c
```

Comment:
Even if the name is different, surely this was heavily inspired from `debug-pl011.inc` in Xen.

If that's correct, I am afraid this will likely have to be fully re-written because the license are not compatible.

Reply:
XTF is BSD-2. Xen is GPL. BSD was not compatible with GPL but BSD-2 is

Reply:
> XTF is BSD-2. Xen is GPL. BSD was not compatible with GPL but BSD-2 is

IANAL. My understanding is it means you can import BSD-2 code in GPL code but not the inverse because GPL requires you to distribute the code with the binary.

Reply:
I've got confirmation that we cannot import GPL code into BSD-2. I will keep it in mind when searching for inspiration :)